sdk/elixir/dev/elixir_sdk_dev/lib/elixir_sdk_dev.ex

```diff
@@ -0,0 +1,42 @@
+defmodule ElixirSdkDev do
+  @moduledoc """
+  A module for help developing Elixir SDK.
+  """
+
+  use Dagger.Mod.Object, name: "ElixirSdkDev"
+
+  @base_image "hexpm/elixir:1.17.2-erlang-27.0.1-alpine-3.20.2@sha256:7c8a13cbff321b7d6f54b4c9a21a10fc8b987974171231eaa77532b8e638b645"
```

Comment:
Obviously up to you as maintainer, but this now changes the behavior to only test on this version, instead of all the `elixirVersions`.

Reply:
Also perhaps jump straight into using [27.1.2](https://github.com/erlang/otp/releases/tag/OTP-27.1.2) & use latest alpine? E.g.:

```
1.17.3-erlang-27.1.2-alpine-3.20.3
```

Reply:
> Obviously up to you as maintainer, but this now changes the behavior to only test on this version, instead of all the `elixirVersions`.

That is my intention. I'm start using new feature in runtime and codegen that requires new version of Erlang/OTP (such as json) and older versions make me spend much more time to fix the issue while my free time to working on the SDK is limited.

Reply:
> Also perhaps jump straight into using [27.1.2](https://github.com/erlang/otp/releases/tag/OTP-27.1.2) & use latest alpine? E.g.:
> 
> 
> 
> ```
> 
> 1.17.3-erlang-27.1.2-alpine-3.20.3
> 
> ```

Agree. I will open a new PR to tackle this one together with the runtime. ✌️